static/app/views/profiling/profileSummary/recentProfilesTable.tsx

```diff
@@ -0,0 +1,102 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import Pagination from 'sentry/components/pagination';
+import {ProfileEventsTable} from 'sentry/components/profiling/profileEventsTable';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useProfileEvents} from 'sentry/utils/profiling/hooks/useProfileEvents';
+import {formatSort} from 'sentry/utils/profiling/hooks/utils';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+
+const BASE_FIELDS = [
+  'profile.id',
+  'trace',
+  'trace.transaction',
+  'profile.duration',
+  'timestamp',
+  'release',
+  'environment',
+  'os.name',
+  'os.version',
+] as const;
+
+// user misery is only available with the profiling-using-transactions feature
+const ALL_FIELDS = [...BASE_FIELDS, 'user_misery()'] as const;
+type FieldType = (typeof ALL_FIELDS)[number];
+
+export function RecentProfilesTable() {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const profilingUsingTransactions = organization.features.includes(
+    'profiling-using-transactions'
+  );
+  const fields = profilingUsingTransactions ? ALL_FIELDS : BASE_FIELDS;
```

Comment:
I would not include `user_misery()` here as it's an aggregation. This table should just be the raw fields.